department-of-veterans-affairs/abd-vro
# MCP 2302: Add date of claim to assessment request message

## What was the problem?

Date of claim submission is collected from the `automatedClaim` endpoint but that date was not making it to the python assessment services. 
In version 1, we default to today's date if there is no date passed in for “dateOfClaim”.
Associated tickets or Slack threads:

- [MCP-2302](https://amida.atlassian.net/browse/MCP-2302)

## How does this fix it?[^1]

Pass along the date of claim to be used in the python assessment service.

## How to test this PR
- Run automatedClaim endpoint with the default body
- Use the evidence-pdf endpoint with "claimSubmissionId" :1234
 There should be blood pressure readings (from 2017, 2018) and medication on the PDF. 

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
